frontend/src/components/modals/pointPopup/pointPopup.scss

```diff
@@ -0,0 +1,29 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
```

Comment:
We don't use scss. Maybe try to look at this https://emotion.sh/docs/globals. And try to replace global scss via global emotion component

Reply:
This is a general style reset for the entire application. I'm not sure if <CssBaseline /> handles this correctly.

Reply:
This is a general style reset for the entire application. I'm not sure if CssBaseline handles this correctly.

Reply:
> This is a general style reset for the entire application. I'm not sure if CssBaseline handles this correctly.

It's okey. I think <Global> component should work to reset

Reply:
scss still here.
https://stackoverflow.com/a/69218520

Reply:
I have reseted all styles via GlobalStyle